### DISCUSSION TITLE
DITA-OT 4.1.2

### DISCUSSION THREAD

**Discussion by infotexture** [dita-ot/dita-ot#4290] [Announcements]
### DITA-OT 4.1.2 has been released

✨  For details on what’s new, see the _Release Notes_ at [dita-ot.org/4.1/release-notes/#v4.1.2](https://www.dita-ot.org/4.1/release-notes/#v4.1.2).

⬇️ The [distribution package](https://github.com/dita-ot/dita-ot/releases/download/4.1.2/dita-ot-4.1.2.zip) is available from the [Downloads](https://www.dita-ot.org/download) page on the project website.

**Comment by chrispy-snps** [comment]
Impressive release!
